### PR TITLE
Procedure: DRAFT RFP Process + Updated readme with basic content

### PR DESCRIPTION
**Pull request by rayraspberry**

### CODE REVIEW

**Comment by rayraspberry** [comment]
I got some of the commits crossed into this PR by accident, I think I've got things resolved.  This PR just effects the two files and the other branches still exist with their individual document updates related to their PRs

File: departments/development/procedures/DRAFT-001-RFP.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Publish the RFP document.
+
+1. Collect and review proposals.
+
+1. Hold Private Meetings to discuss proposals and determine if the submissions are qualified.
+    1. Notify Qualified Candidates and request their acceptance to schedule a public meeting with candidates.
+
+1. Publish a list of qualified candidates with the candidate's proposal.
+    1. Publish a list of qualified candidates with the candidate's proposal.
```
Comment (kopeboy) [inline departments/development/procedures/DRAFT-001-RFP.md]: repeated line

File: departments/development/procedures/DRAFT-001-RFP.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    1. Notify all candidates of their status.
+
+1. Hold a Public Meeting with each candidate to discuss their proposal.
+    1. Candidate presents proposal publicly
+    1. Delegates & Public ask questions
+    1. Define conditions and requirements for each proposal.
```
Comment (kopeboy) [inline departments/development/procedures/DRAFT-001-RFP.md]: I would move the last sub-point under the next point, or change it to something like "Gather additional requirements and conditions".

File: departments/development/procedures/DRAFT-001-RFP.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# PROCEDURE:
+
+## Request for Proposal (RFP) Creation and Management
+
+1. Define the scope of work and requirements for the RFP.
+
+1. Create a draft RFP document.
+    1. Use Development Department RFP Template
+    1. Publish an to Github
```
Comment (kopeboy) [inline departments/development/procedures/DRAFT-001-RFP.md]: ~~an~~ a PR? or`Publish it to Github with a Pull Request`.

**Pull request merged**